README.md

```diff
@@ -1,15 +1,28 @@
-# Caladrius - Assessing Building Damage caused by Natural Disasters using Satellite Images
+# [Caladrius](https://en.wikipedia.org/wiki/Caladrius) - Assessing Building Damage caused by Natural Disasters using Satellite Images
 ## Created by: Artificial Incompetence for the Red Cross #1 Challenge in the 2018 Hackathon for Peace, Justice and Security
 
 ## Network Architecture
 
 The network architecture is a pseudo-siamese network with two ImageNet pre-trained Inception_v3 models.
 
+## Using Docker
 
-## Setup
+Install [Docker](https://www.docker.com/get-started).
+
+Download the [Caladrius Docker Image](https://hub.docker.com/r/gulfaraz/caladrius) using `docker pull gulfaraz/caladrius`.
+
+Create a [data](#dataset) folder in your local machine.
+
+Create a docker container using `docker run --name caladrius -dit -v <path/to/data>:/workspace/data gulfaraz/caladrius`.
```

Comment:
To create docker container, should it be -it instead of -dit?

Reply:
with `-d` the container is **detached** and keeps running in the *background*.

So one can **login** and **logout** of the container like a *virtual machine*.

I chose to use `-d` to separate **creation** and **access** commands.

Let me know if you feel this is too inconvenient.